pkg/process/config/config.go

```diff
@@ -425,24 +392,15 @@ func loadEnvVariables() {
 		{"DD_STRIP_PROCESS_ARGS", "process_config.strip_proc_arguments"},
 		{"DD_PROCESS_AGENT_URL", "process_config.process_dd_url"},
 		{"DD_PROCESS_AGENT_INTERNAL_PROFILING_ENABLED", "process_config.internal_profiling.enabled"},
-		{"DD_PROCESS_AGENT_REMOTE_TAGGER", "process_config.remote_tagger"},
```

Comment:
Should we add `DD_PROCESS_AGENT_REMOTE_TAGGER` and `DD_PROCESS_CONFIG_REMOTE_TAGGER` to the QA section on the PR description as well?

Reply:
Oh yeah, that's a good point! I didn't think of that